packages/scripts/entrypoint_nestri.sh

```
@@ -156,8 +149,8 @@
         NESTRI_LAUNCH_COMPOSITOR="gamescope --backend wayland --force-grab-cursor -g -f --rt --mangoapp -W ${WIDTH} -H ${HEIGHT} -r ${FRAMERATE:-60}"
     fi
 
-    # Start Steam patcher only if Steam command is present
-    if [[ -n "${NESTRI_LAUNCH_CMD}" ]] && [[ "$NESTRI_LAUNCH_CMD" == *"steam"* ]]; then
+    # Start Steam patcher only if Steam command is present and running under non-sane container engines
+    if [[ -n "${NESTRI_LAUNCH_CMD}" ]] && [[ "$NESTRI_LAUNCH_CMD" == *"steam"* ]] && [[ "$container_runtime" != "podman" ]]; then
```

Shellcheck (0.10.0)

[warning] container_runtime is referenced but not assigned.

(SC2154)